disconnected/updating/mirroring-image-repository.adoc

```diff
@@ -39,6 +39,11 @@ The following steps outline the high-level workflow on how to mirror images to a
 
 .. Repeat these steps as needed to update your mirror registry.
 
+[IMPORTANT]
+====
+The `oc adm catalog mirror` and `oc adm release mirror` commands are being deprecated as their features move to `oc-mirror`. These commands will remain in the `oc` codebase but will be marked as deprecated until `oc-mirror` is fully implemented and tested.
```

Comment:
Suggested reword:
```suggestion
The `oc adm catalog mirror` and `oc adm release mirror` commands have been deprecated and will be removed in a future release.
Their features are planned to be included in the oc-mirror OpenShift CLI (`oc`) plugin.
```

I think it's probably important to say something like "planned to be included" so as to not formally commit to anything in a future release, no matter how safe that assumption is.

Reply:
@skopacz1,

In order to say "planned to be included" it is necessary first to list all features available in the `oc adm catalog mirror` and `oc adm release mirror` and talk with the oc-mirror team to see if they are planned to be included.

cc: @lmzuccarelli @sherine-k

Reply:
Should this notice just focus on the deprecation of these commands, then? Or does this notice need to give some sort of assurance to users that eventually some features will migrate to oc-mirror?

I feel like usually deprecation notices in our docs just say "feature x is deprecated" and leave it at that, and that any announcement of feature migration could be mentioned in a release note. But I also don't have the full context for this deprecation.

Reply:
@skopacz1 I do not have the full context about this deprecation to answer your question. Probably it is better to check with the workloads team (responsible for both commands being deprecated)

Reply:
As per the latest discussion with the team, we can halt this PR and work for a solution after the GA.